src/platforms/flutter/troubleshooting.mdx

```diff
@@ -23,3 +23,9 @@ Building a Flutter app for a specific ABI using the `--target-platform` argument
 ```bash
 flutter build apk --target-platform=android-arm --split-per-abi
 ```
+
+## Native symbolication on Android
+
+For the symbolication of Native events (C/C++) on Android, you've to make a configuration which is described on the [Android Native Development Kit](/platforms/android/using-ndk/) page.
+
+Specifically about `android:extractNativeLibs="true"` in the `AndroidManifest.xml` or `android.bundle.enableUncompressedNativeLibs=false` in the `gradle.properties`, otherwise you might not have a good symbolication experience.
```

Comment:
I think we can remove this here, as it is described in `/platforms/android/using-ndk/`.

Reply:
that's the problem we're trying to solve, so people don't need to read everything and get lost, so I'd say this is actually necessary, the debug symbols page already links to the page above, but people were still missing it

Reply:
I've made some edits to the wording to make it clearer, but @PeloWriter might still want to review this addition.

Reply:
thanks :)